### ISSUE TITLE
Pull Requests links don't work

### ISSUE THREAD

**Issue by baileyjs78** [ostreifel/Pull-Request-Search#10]
I am using TFS On Prem 2017, and latest version 1.13.3
When I click on the links 2 things happen
1) The pull request details are not shown (due to auth issues)
2) The authentication session is lost (confirmed by going to backlog)
I have confirmed this in latest Firefox and Chrome
The links look to be correct.
If I open the link in a new tab, or copy the url and paste it, i have no problems
However, if I actually click the link it fails

Another thing I noticed is that on the pull request search page, it is showing an error trying to access (image from firebug)
/_Bundling/DynamicBundles?scripts=VSS%2FAuthentication%2FContracts%3B*RestClient&v=Dev15.M105.5&json

![pullrequesterror3](https://cloud.githubusercontent.com/assets/26547339/24208391/e5c76196-0ef9-11e7-9948-99b5f48d8026.jpg)
![pullrequesterror2](https://cloud.githubusercontent.com/assets/26547339/24208389/e5bedb02-0ef9-11e7-8625-35eba13132b2.jpg)
![pullrequesterror1](https://cloud.githubusercontent.com/assets/26547339/24208390/e5c365d2-0ef9-11e7-8d98-38092bff4acb.jpg)

**Comment by ostreifel** [comment]
Tried this with tfs 2017 (verison 15.105.25910.0) on ie, edge, firefox and chrome with no repro.

If you remove the rel attribute from the link in firebug does error still repro?

**Comment by baileyjs78** [comment]
I tied removing the rel attribute and it still repro
I tried adding "#_a=overview" to the URL (that is what I see in TFS) and it still repro
I tried both, and it still repro

**Comment by ostreifel** [comment]
I know you tested this on firefox/chrome. 
Does this also repo on 
- private browing modes of firefox/chrome
- edge
- ie
- your phone
- another user account of the same server?

I was using a clean install tfs 2017 when I tried to repo I and could not. Are there additional configurations that need to be done to the server for this to repo?

**Comment by baileyjs78** [comment]
I tried another user, browser and private mode and it still reproduced

We have multiple teams and multiple project collections. That is the only other thing I can think of.
I disabled all the other extensions that I could, and it still reproduced
This only left me with
7 Zip (used in builds)
Code Search
MSdeployAllTheThings (used in builds)
On-Premises Release toolkit (used in builds)
Package Management
Redgate ReadyRoll (used in builds)
Web.Config Tools (used in builds)
WhiteSource (used in builds)

I have attached a recording on the process


Jeremy Bailey

Applications Development Manager
315.793.4900, ext. 2225
